modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/apirequest/ApiRequestEnrichment.scala

```diff
@@ -212,43 +202,28 @@ sealed trait CreateApiRequestEnrichment[F[_]] {
 object CreateApiRequestEnrichment {
   def apply[F[_]](implicit ev: CreateApiRequestEnrichment[F]): CreateApiRequestEnrichment[F] = ev
 
-  implicit def idCreateApiRequestEnrichment(
-    implicit CLM: CreateLruMap[Id, String, (Either[Throwable, Json], Long)],
-    HTTP: HttpClient[Id]
-  ): CreateApiRequestEnrichment[Id] =
-    new CreateApiRequestEnrichment[Id] {
-      override def create(conf: ApiRequestConf): Id[ApiRequestEnrichment[Id]] =
-        CLM
-          .create(conf.cache.size)
-          .map(c =>
-            ApiRequestEnrichment(
-              conf.schemaKey,
-              conf.inputs,
-              conf.api,
-              conf.outputs,
-              conf.cache.ttl,
-              c
-            )
-          )
-    }
-
-  implicit def syncCreateApiRequestEnrichment[F[_]: Sync](
-    implicit CLM: CreateLruMap[F, String, (Either[Throwable, Json], Long)],
-    HTTP: HttpClient[F]
+  implicit def instance[F[_]: Monad: HttpClient: Clock](
+    implicit CLM: CreateLruMap[F, String, CachingEvaluator.CachedItem[Json]]
   ): CreateApiRequestEnrichment[F] =
     new CreateApiRequestEnrichment[F] {
-      def create(conf: ApiRequestConf): F[ApiRequestEnrichment[F]] =
-        CLM
-          .create(conf.cache.size)
-          .map(c =>
+      def create(conf: ApiRequestConf): F[ApiRequestEnrichment[F]] = {
+        val cacheConfig = CachingEvaluator.Config(
+          size = conf.cache.size,
+          successTtl = conf.cache.ttl,
+          errorTtl = conf.cache.ttl / 10
```

Comment:
Wondering if we should make error ttl configurable next to the 'success' ttl or just do it like this, so divide 'success' by some magic number?